nilearn/_utils/docs.py

```diff
@@ -163,6 +191,22 @@
     are .png, .pdf, .svg. If ``output_file`` is not None, the plot
     is saved to a file, and the display is closed."""
 
+# extractor / extract_type
+docdict['extractor'] = """
+extractor : :obj:`str` {'local_regions', 'connected_components'}, optional
```

Comment:
I think numpydoc convention is just to have the options, without the data type. See [here](https://numpydoc.readthedocs.io/en/latest/format.html):

>When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:
>
> ```
> order : {'C', 'F', 'A'}
>     Description of `order`.
> ```